packages/ui/components/apps/AppCard.tsx

```diff
@@ -201,6 +205,19 @@ const InstallAppButtonChild = ({
   });
 
   if (!userAdminTeams?.length || !doesAppSupportTeamInstall(appCategories, concurrentMeetings)) {
```

Comment:
We need to fix this check. Currently I am logged in as an admin/owner of an org and `useAdminTeams` has certain items in it. So, first condition is false.

Second condition though I was expecting it to be true is false because by `doesAppSupportTeamInstall`'s logic any app that is not a calendar or video app supports team installation.


I suggest disabling the team installation for any paid app because we haven't implemented that yet(I guess). So, we could just move line 208 - 219 out of this if condition

Reply:
I have fixed it as suggested after verifying that cal.ai doesn't support team install anyways.
https://github.com/calcom/cal.com/blob/5ef737f22ecb9a2fdbc5c11f9883f6b241dda6a4/apps/ai/src/app/api/onboard/route.ts#L11